ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -180,124 +143,85 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
   };
 
   return (
-    <LoadingSpinnerOverlay
-      isLoading={
-        props.isCgnInfoLoading ||
-        (props.cgnDetails && cardLoading) ||
-        isLoading(props.unsubscriptionStatus)
+    <BonusCardScreenComponent
+      logoUris={logoUris}
+      name={I18n.t("bonus.cgn.name")}
+      organizationName={I18n.t("bonus.cgn.departmentName")}
+      counters={[]}
+      cardBackground={<AnimatedBackground />}
+      status={
+        props.cgnDetails ? <CgnCardStatus card={props.cgnDetails} /> : undefined
+      }
+      cardFooter={
+        <H2
```

Comment:
From the designed figma, the name of the user has to be in an `H4` and not in an `H2`

Reply:
How can I check this? For future reference thank you.